shell/platform/tizen/tizen_surface_gl.cc

```diff
@@ -77,14 +78,127 @@ uint32_t TizenSurfaceGL::OnGetFBO() {
   return 0;  // FBO0
 }
 
+#define GL_FUNC(FunctionName)              \
+  else if (strcmp(name, #FunctionName) == 0) {    \
+    return reinterpret_cast<void*>(FunctionName); \
+  }
 void* TizenSurfaceGL::OnProcResolver(const char* name) {
   auto address = eglGetProcAddress(name);
   if (address != nullptr) {
     return reinterpret_cast<void*>(address);
   }
+  GL_FUNC(eglGetCurrentDisplay)
```

Comment:
I think hardcode is not the best way...

Reply:
I think so too...but.. I don't have any ideas right now.